<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetDrag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="addButtonsLayout">
     <item>
      <widget class="QPushButton" name="addByteButton">
       <property name="text">
        <string>Add Byte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addNumberButton">
       <property name="text">
        <string>Add Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addVectorButton">
       <property name="text">
        <string>Add Vector</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printListButton">
       <property name="text">
        <string>Print List</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
